ext/opentelemetry-ext-opentracing-shim/tests/test_shim.py

```diff
@@ -63,7 +63,7 @@ def test_shim_type(self):
     def test_start_active_span(self):
         """Test span creation and activation using `start_active_span()`."""
 
-        with self.shim.start_active_span("TestSpan") as scope:
+        with self.shim.start_active_span("TestSpan0") as scope:
```

Comment:
was there a reason for changing the name of the spans here? i'm not seeing a test that validates the name is correct anywhere

Reply:
Yes, it makes it easier to debug things, in this way I was able to tell which span was left unfinished from a previously executed test case.